soh/src/overlays/actors/ovl_Boss_Goma/z_boss_goma.c

```diff
@@ -326,6 +326,7 @@ void BossGoma_ClearPixels32x32Rgba16(s16* rgba16image, u8* clearPixelTable, s16
  * Clear pixels from Gohma's textures
  */
 void BossGoma_ClearPixels(u8* clearPixelTable, s16 i) {
+    return;
```

Comment:
Can this be fixed somehow?

Reply:
It's actually the `BossGoma_ClearPixels32x32Rgba16` method that's crashing. I think we should be able to rewrite that so it also works on 64bit machines